src/cmake/thirdparty/SetupAxomThirdParty.cmake

```diff
@@ -140,10 +141,10 @@ endif()
 #------------------------------------------------------------------------------
 if (SCR_DIR)
     include(cmake/thirdparty/FindSCR.cmake)
-    blt_register_library( NAME      scr
-                          INCLUDES  ${SCR_INCLUDE_DIRS}
-                          LIBRARIES ${SCR_LIBRARY}
-                          TREAT_INCLUDES_AS_SYSTEM ON)
+    blt_import_library( NAME      scr
+                        INCLUDES  ${SCR_INCLUDE_DIRS}
+                        LIBRARIES ${SCR_LIBRARY}
+                        TREAT_INCLUDES_AS_SYSTEM ON)
```

Comment:
Does this imply that we're not exporting `scr`?

Reply:
I had missed this one - all the TPLs here that previously used `blt_register_library` now use `blt_import_library` + `EXPORTABLE`.